jest.config.js

```diff
@@ -1,5 +1,9 @@
 module.exports = {
   preset: 'ts-jest',
   testEnvironment: 'node',
-  modulePathIgnorePatterns: ['<rootDir>/lib', '<rootDir>/bin', '<rootDir>/tmp', '<rootDir>/test/nest', '<rootDir>/test/express']
+  modulePathIgnorePatterns: ['<rootDir>/lib', '<rootDir>/bin', '<rootDir>/tmp', '<rootDir>/test/nest', '<rootDir>/test/express'],
+  transform: {"^.+\\.tsx?$": "ts-jest"},
+  testMatch: ["**/*.spec.ts"],
+  roots: ["test"],
+  reporters: ['default']
```

Comment:
Is this change necessary? Seems like it's all the default values and can be omitted

Reply:
they are almost not necessary, only the root so that the sample test files in templates are not executed.